keyboards/ergotravel/keymaps/xyverz/keymap.c

```diff
@@ -0,0 +1,134 @@
+﻿#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _DVORAK 0
+#define _QWERTY 1
+#define _COLEMAK 2
+#define _LOWER 5
+#define _RAISE 6
+#define _ADJUST 16
+
+enum ergotravel_keycodes {
+  DVORAK = SAFE_RANGE,
+  QWERTY,
+  COLEMAK,
+  LOWER,
+  RAISE
+};
+
+// Fillers to make layering more clear
+#define _______ KC_TRNS
```

Comment:
Both these aliases are defined in the QMK core; they're not needed here.

Lines 24-26 should be deleted.

https://github.com/qmk/qmk_firmware/blob/8fd3f42281885346f93fff2f122569c002071c67/quantum/quantum_keycodes.h#L29-L31

Reply:
Good to know, thanks!